vcf_zarr_spec.md

```diff
@@ -0,0 +1,122 @@
+# VCF Zarr specification
+
+***Version 0.1***
+
+This document is a technical specification for VCF Zarr, an equivalent of VCF that uses Zarr storage.
```

Comment:
```suggestion
This document is a technical specification for VCF Zarr, a means of encoding VCF data in chunked-columnar form using the Zarr format.
```
I thought it good to give some more concrete details here? This is just a suggestion, my wording isn't great.